examples/constants.js

```diff
@@ -0,0 +1,477 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* eslint-disable no-template-curly-in-string */
+
+const MARKET_ITEM_LIST = [
+  {
+    id: '0b41d10e-36e5-4e97-95c2-27726dd94f33',
+    name: 'COVID-19 image data collection',
+    author: 'ieee8023',
+    type: 'data',
```

Comment:
Could you add dataUrl and dataType values after the new download feature merged?

Reply:
OK, will add after the download PR merge

Reply:
Done.